ciphers/rsa/RSAcipher.go

```diff
@@ -1,11 +1,13 @@
-package main
+package rsacipher
```

Comment:
This package fails the test for the encryption and decryption - I though it was failing because the implementation of modular exponentiation was incorrect but the test for modular exponentiation passes without issues so I'm actually not sure what the problem is. My current theory is that the inverse exponentiation function is incorrect but I did't try to test it.